tests/conftest.py

```diff
@@ -586,8 +586,9 @@ def create_client(
         inited_client = weave_init.init_weave("dev_testing")
 
     if inited_client is None:
-        # This is disabled by default, but we explicitly enable it here for testing
-        os.environ["WEAVE_USE_SERVER_CACHE"] = "true"
+        # Removing this as it lead to passing tests that were not passing in prod!
```

Comment:
This was enabled for a while with the intention of showing that it would work in all scenarios, but in a testing context, this hides problems. Disabling for now.